<launch>
  <arg name="droneip" default="192.168.1.1" />
  <node name="ardrone_driver" pkg="ardrone_autonomy" type="ardrone_driver" output="screen" args="-ip $(arg droneip)">
    <param name="navdata_demo" value="False" />
    <param name="realtime_navdata" value="True" />
    <param name="realtime_video" value="True" />
    <param name="looprate" value="30" />
    <param name="outdoor" value="False" />
    <param name="flight_without_shell" value="True" />
    <param name="altitude_max" value="2000" />
    <param name="altitude_min" value="1500" />
    <param name="control_vz_max" value="2500" />
    <param name="control_yaw" value="700" />
  </node>
</launch>
